dmsnitch/resource_snitch.go

```diff
@@ -82,6 +91,12 @@ func resourceSnitch() *schema.Resource {
 }
 
 func newSnitchFromResource(d *schema.ResourceData) *Snitch {
+	email := make([]string, 0, len(d.Get("alert_email").(*schema.Set).List()))
```

Comment:
```suggestion
	emails := make([]string, 0, len(d.Get("alert_emails").(*schema.Set).List()))
```
I'd prefer this var be named plural to reflect the fact that it is an array

Reply:
I agree but the field in the json from their API is alert_email. That is why I kept the naming consistent with how they named it.  Do you want a change to plural everywhere except the actual field or leave it how I originally had it?